config/config.yaml

```diff
@@ -3,6 +3,8 @@ samples: "samples.tsv"
 units: "units.tsv"
 output: "config/output_files.json"
 
+aligner: "bwa_gpu" # bwa_gpu or bwa_sentieon
```

Comment:
actually i think we need to add this in .tests/integration/config.yaml file

Reply:
solved in @ 00cb0ea